lib/tasks/jobs.rake

```diff
@@ -49,4 +49,11 @@ namespace :jobs do
       FacetLinkGroupGeneratorJob.perform_later facet_link_group
     end
   end
+
+  desc 'Queue GalleryValidationJob'
+  task gallery_validation: :enironment do
+    Gallery.published.each do |gallery|
+      GalleryValidationJobJob.perform_later(gallery.id)
```

Comment:
`GalleryValidationJob` not `GalleryValidationJobJob`

Reply:
good jobjob you spotted this